Gemfile

```diff
@@ -81,3 +81,15 @@ end
 
 gem "govuk_notify_rails", "~> 2.2", ">= 2.2.0"
 
+gem 'sass-rails'
```

Comment:
I'd hold off adding `sass-rails` at the moment, as it causing problems elsewhere